rstudio/c9s-python-3.9/Dockerfile

```diff
@@ -48,7 +48,8 @@ RUN chmod 1777 /var/run/rstudio-server && \
 COPY rsession.conf /etc/rstudio/rsession.conf
 
 # package installation 
-RUN dnf install -y libsodium-devel.x86_64 libgit2-devel.x86_64 libcurl-devel harfbuzz-devel.x86_64 fribidi-devel.x86_64 cmake "flexiblas-*"
+RUN dnf install -y libsodium-devel.x86_64 libgit2-devel.x86_64 libcurl-devel harfbuzz-devel.x86_64 fribidi-devel.x86_64 cmake "flexiblas-*" \
```

Comment:
Thank you, Jan! These changes look good to me. I tested the RStudio notebook locally with the new changes recommended by the `hadolint` action, and everything appears to be working well without any issues. The RStudio image I used is `quay.io/rh_ee_atheodor/workbench-images:rstudio-c9s-python-3.9-2023b_20240226`. Additionally, I noticed that the size of the image has been reduced by a few megabytes due to the `dnf clean all && rm -rf /var/cache/yum` , which is great!
/lgtm